### PR TITLE
Site: Remove '(for Calcite committers)' suffix from headers in section dedicated to committers

### PR DESCRIPTION
**Pull request by zabetak**

### CODE REVIEW

File: site/_docs/howto.md [hunk 1]
```diff
@@ -474,7 +474,7 @@ $ ./src/main/scripts/generate-protobuf.sh
 The following sections are of interest to Calcite committers and in
 particular release managers.
 
-## Managing Calcite repositories through GitHub (for Calcite committers)
+## Managing Calcite repositories through GitHub
 
 Committers have write access to Calcite's 
 [ASF git repositories](https://gitbox.apache.org/repos/asf#calcite) hosting
```
Comment (chunweilei) [inline site/_docs/howto.md:480]: Why should we remove the suffix?

Comment (zabetak) [reply site/_docs/howto.md:480]: The whole section is about committers (see line 472) and the suffix is repeated in every subsection so IMHO it is redundant. It makes headers and links longer without adding new information.

Comment (zinking) [reply site/_docs/howto.md:480]: > The whole section is about committers (see line 472) and the suffix is repeated in every subsection so IMHO it is redundant. It makes headers and links longer without adding new information.

-1, I'm not convinced.

Comment (rubenada) [reply site/_docs/howto.md:480]: I think @zabetak has a point. If we have a section called "Advanced topics for committers", then it seems redundant to have every subsection in there called "XYZ (for Calcite committers)".
Right before that we have an "Advanced topics for developers" section and we do not have the suffix "(for developers)" on each subsection.

Comment (chunweilei) [reply site/_docs/howto.md:480]: > The whole section is about committers (see line 472) and the suffix is repeated in every subsection so IMHO it is redundant. It makes headers and links longer without adding new information.

Makes sense. Thanks for your explanation.

**Comment by rubenada** [comment]
LGTM

**Comment by zabetak** [comment]
Thanks everybody for your feedback. Given that there is a -1, I cannot move this forward. @zinking I don't want to start a long exchange about such a minor issue but I would like to know why do you think its not an improvement. Can you please elaborate a bit more?

**Comment by zinking** [comment]
> Thanks everybody for your feedback. Given that there is a -1, I cannot move this forward. @zinking I don't want to start a long exchange about such a minor issue but I would like to know why do you think its not an improvement. Can you please elaborate a bit more?

like you said, it's such a minor change. and I'm good with ruben's explaination now. please move forward.

**Pull request merged**